api/docs/developer-system-initialization.md

```diff
@@ -97,10 +97,18 @@ git secrets --scan /path/to/file (/other/path/to/file *)
 git secrets --scan -r /path/to/directory
 ```
 
+## envsubst
+
+`envsubst` is part of the `gettext` package.
```

Comment:
This is needed by some `dev-up` step

Reply:
I think this install cmd probably is more appropriate in `developer-system-requirements.md`

Reply:
done